### ISSUE TITLE
[sec] Email fields at /tipsy.txt 

### ISSUE THREAD

**Issue by rht** [haystack/tipsy#90]
The email fields at /tipsy.txt are crawlable by bots. Either encode it (a bit cumbersome for end-users) or ...

**Comment by da2x** [comment]
This isn't really much of a problem. Just be sure to ask publishers to use dedicated email addresses. PayPal requires business accounts to receive service payments anyway, so websits should already be using addresses like `tipsy-micropayments@mydomain.com` for this purpose (and bookkeeping).

The only bug here is that the documentation should be improved.

**Comment by da2x** [comment]
You can also use your merchant account ID instead of an email address.

Note that a merchant account is required when you receive payment for business purposes as per PayPal’s terms of stuff. Forms and documentation should be updated. The generator script should have it's valid email formatting requirement removed, etc.

**Comment by rht** [comment]
Right, I see -- it looks like the merchant account ID (in the context of paypal's protocol, and possibly there is an equivalent in dwolla's as well) was devised to replace email address for the button code. This is a breaking change, and could be coordinated with the priv key migration to fix #101.
